backend/src/app/rpc/queries/files.clj

```diff
@@ -420,7 +421,28 @@
 
 (sv/defmethod ::team-shared-files
   [{:keys [pool] :as cfg} {:keys [team-id] :as params}]
-  (db/exec! pool [sql:team-shared-files team-id]))
+  (let [assets-sample
+        (fn [assets limit]
+          (let [sorted-assets (->> assets
+                                   (vals)
+                                   (vec)
+                                   (sort-by #(str/lower (:name %))))]
```

Comment:
Can be simplified with:

```clojure
(->> (vals assets)
     (sort-by (comp str/lower :name)))
```
You don't need call `vec` here.

Reply:
ok